types/carbon__pictograms-react/index.d.ts

```diff
@@ -0,0 +1,466 @@
+// Type definitions for @carbon/pictograms-react 10.7
+// Project: https://github.com/carbon-design-system/carbon/tree/master/packages/pictograms-react
+// Definitions by: Eric Liu <https://github.com/metonym>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.5
+
+export interface CarbonPictogramProps extends Omit<React.SVGProps<any>, 'tabIndex'> {
```

Comment:
Would `SVGElement` be a better type argument for SVGProps?

Reply:
Yes, will add.